content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -639,3 +639,79 @@ Don't forget to clean up the debugging Pod when you're finished with it:
 ```shell
 kubectl delete pod node-debugger-mynode-pdx84
 ```
+
+## Debugging Profiles {#debugging-profiles}
+
+When you use `kubectl debug` for debugging a node, a Pod by ephemeral container, or copy, you can apply a debugging profile to them via `--profile` flag.
+By applying a profile, specific properties such as [securityContexts](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) are set to them, allowing for adaptation to various scenarios.
```

Comment:
@tengqm 

> Question: what does "them" refer to?

It indicates "a node, a Pod by ephemeral container, or copy"

Reply:
Okay, that is a bit confusing to me.
If the "them" refers to the node, pod (copy) that is to be debugged,
then, the `securityContext` for "them" are adapted by applying a profile.
According to this statement, if 'baseline' profile is specified for a node debug session,
the `securityContext` of the Node is changed. I don't think that is correct.

Reply:
@tengqm 

Oh, I see!
I've rethought the sentence.
How about that.

Reply:
Sorry I didn't see any changes related to the usage of "them".
There are two "them" in this paragraph.
Both of them are confusing.

Reply:
@tengqm 
I've fix that

> When you use `kubectl debug` for debugging a node, a Pod by ephemeral container, or copy, 

->
 
> When using `kubectl debug` to debug a node via a debugging Pod, a Pod via an ephemeral container, 
or a copied Pod,

By this change, I intended to indicate that each "them" refers to "a debugging Pod, a Pod via an ephemeral container, or a copied Pod" not Node itself.

> Both of them are confusing.

Should I explicitly indicate "Pod or container" instead of using "them"?

Reply:
I've rethought and removed second "them" as below:

> By applying a profile, specific properties such as [securityContext](/docs/tasks/configure-pod-container/security-context/)
are set, allowing for adaptation to various scenarios.